src/components/select/option-group-header/option-group-header.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import OptionGroupHeader from ".";
+import guid from "../../../__internal__/utils/helpers/guid";
+
+const mockedGuid = "guid-12345";
+jest.mock("../../../__internal__/utils/helpers/guid");
+(guid as jest.MockedFunction<typeof guid>).mockImplementation(() => mockedGuid);
```

Comment:
**question**: it doesn't appear we are asserting the id set on an underlying element in any of the tests. Is this mock required for something else?

Reply:
I'll look into it, just left them as they were in the original files